pom.xml

```diff
@@ -14,14 +15,17 @@
     <parent>
         <groupId>org.springframework.data.build</groupId>
         <artifactId>spring-data-parent</artifactId>
-        <version>2.6.0-SNAPSHOT</version>
+        <version>3.0.0-SNAPSHOT</version>
     </parent>
 
     <properties>
-        <couchbase>3.2.1</couchbase>
-        <couchbase.osgi>3.2.1</couchbase.osgi>
-        <springdata.commons>2.6.0-SNAPSHOT</springdata.commons>
+        <couchbase>3.2.3</couchbase>
+        <couchbase.osgi>3.2.3</couchbase.osgi>
+        <springdata.commons>3.0.0-SNAPSHOT</springdata.commons>
         <java-module-name>spring.data.couchbase</java-module-name>
+        <jodatime>2.10.13</jodatime>
```

Comment:
We've dropped support for Jodatime in Spring Data Commons. Unless there's a strong reason to keep Joda, we suggest switching to JSR-310 types.

Reply:
We have this as optional - if the user provides the jodatime then we will support it.  I believe the JSR-310 types are handled by https://github.com/spring-projects/spring-data-couchbase/blob/5.0.x/src/main/java/org/springframework/data/couchbase/core/convert/CouchbaseJsr310Converters.java